testsupport/tiers/checks.go

```diff
@@ -1242,6 +1242,10 @@ func networkPolicyIngressFromPolicyGroup(name, group string) namespaceObjectsChe
 	return assertNetworkPolicyIngressForNamespaces(name, metav1.LabelSelector{}, "network.openshift.io/policy-group", group)
 }
 
+func networkPolicyIngressAllowFromDevSandboxPolicyGroup() namespaceObjectsCheck {
+	return assertNetworkPolicyIngressForNamespaces("allow-from-dev-sandbox-managed-ns", metav1.LabelSelector{}, " dev-sandbox/policy-group", "ingress")
+}
```

Comment:
this should be included in the `commonNetworkPolicyChecks` to execute the check for all tiers automatically
https://github.com/codeready-toolchain/toolchain-e2e/blob/67733d28f5757059432c7625a44946e90322d6bc/testsupport/tiers/checks.go#L377-L385

Reply:
got it, thanks for pointing that out! Done in https://github.com/codeready-toolchain/toolchain-e2e/pull/1086/commits/4f54f926aa254b7a0dd14f8487ebd6f404606791